app/src/main/java/ch/epfl/sdp/appart/ScrollingActivity.java

```diff
@@ -45,16 +52,44 @@
     @Inject
     DatabaseService database;
 
+    @Inject
+    LoginService loginService;
+
+    @Inject
+    LocalDatabaseService localDatabaseService;
+
```

Comment:
Since ScrollingActivity extends ToolbarActivity you should be able to access loginService and localDatabaseService without having to define them here too, right? Or are there injection issues with hilt if we don't inject here?

Reply:
I will try this.

Reply:
Ok this works.